css-view-transitions-2/Overview.bs

```diff
@@ -139,13 +139,46 @@ See the <a href="#lifecycle">lifecycle</a> section for more details.
 
 ### Waiting for the new state to stabilize ### {#waiting-for-stable-state}
 In same-document view transitions, the author can indicate when the new state of the transition is in a stable state by using the callback passed to {{Document/startViewTransition}}.
-Since cross-document view transitions are declarative, there is no such explicit promise. Instead, the user agent relies on the [=render-blocking mechanism=] to decide when the stable state is ready.
+Since cross-document view transitions are declarative, there is no such explicit promise. Instead, the user agent relies on the [=render-blocking mechanism=] to decide when the document has reached a stable state.
 In this way, the author can use the the <code data-x="">blocking</code> attribute, to delay the transition until:
 	* All expected scripts are executed, by using the script's {{HTMLScriptElement/blocking}} attribute on required scripts.
 	* All expected styles are executed, by using the style or link's {{HTMLStyleElement/blocking}} attribute on required styles.
 	* All expected HTML elements are seen by the parser, using an <{link/rel/expect}> {{HTMLLinkElement}} element.
 
+Note: overusing the render-blocking mechanism could make it so that the old state remains frozen for a long time, resulting in a jarring experience.
```

Comment:
```suggestion
Note: overusing the render-blocking mechanism could make it so that the old state remains frozen for a long time, resulting in a jarring user experience.
```